locales/en/translation.json

```
@@ -8,13 +8,13 @@
   "We didn't find a page that matches the address you navigated to.": "We didn't find a page that matches the address you navigated to.",
   "Take me home": "Take me home",
   "Fleet not found": "Fleet not found",
-  "We couldn&apos;t find the fleet with id <1>{id}</1>": "We couldn&apos;t find the fleet with id <1>{id}</1>",
+  "We couldn not find the fleet with id <1>{id}</1>": "We couldn not find the fleet with id <1>{id}</1>",
```

There's a couple typos of `couldn not`
```suggestion
  "We could not find the fleet with id <1>{id}</1>": "We could not find the fleet with id <1>{id}</1>",
```

---

great catch. Fixed.